kumascript_settings_local.json-dist

```diff
@@ -1,31 +0,0 @@
-{
```

Comment:
We're still using ``kumascript_settings_local.json`` in SCL3 for configuring KumaScript, so I want the ignore as long as we're in that environment. I'll adjust the ignores and add a TODO comment.